src/PhpSpec/Message/Fatal.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Message;
+
+class Fatal implements MessageInterface {
```

Comment:
a 'Fatal` `Message` doesn't sound like 'the object that knows which example we are in' to me. This needs to be renamed.